server/config/sessions/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  MONGO: require('./mongo'),
```

Comment:
Ah, sorry, didn't realize that was es6 specific syntax. You can always still do:

```
sessions = {}
sessions[DB_TYPES.MONGO] = require('./mongo);
...
module.exports = sessions;
```

I took a brief look at getting babel working on the server yesterday and it seemed fine in development, but haven't looked at precompilation for production yet. Maybe I can get something basic working tonight